e2e/cypress/tests/integration/messaging/emoji_recently_used_spec.js

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// [#] indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @messaging
+
+import timeouts from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    let offTopicPath;
+    let testTeam;
+    const defaultEmojis = ['+1 emoji', 'grinning emoji', 'white check mark emoji'];
+
+    before(() => {
+        // # Login as test user and visit off-topic
+        cy.apiInitSetup({loginAfter: true}).then(({offTopicUrl, team}) => {
+            testTeam = team;
+            offTopicPath = offTopicUrl;
+            cy.visit(offTopicUrl);
+        });
+    });
+
+    it('MM-T4261_1 One-click reactions on a post', () => {
+        // # Ensure emoji picker is enabled in System console
+        cy.apiAdminLogin();
+        cy.visit('/admin_console/site_config/emoji');
+        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Emoji');
+
+        cy.findByTestId('ServiceSettings.EnableEmojiPickertrue').check();
+
+        cy.get('#saveSetting').then((btn) => {
+            if (btn.is(':enabled')) {
+                btn.click();
+
+                cy.waitUntil(() => cy.get('#saveSetting').then((el) => {
+                    return el[0].innerText === 'Save';
+                }));
+            }
+        });
```

Comment:
I'm good as is since @furqanmlk will soon update all similar UI action in saving config as custom command -- likely will be part of `cy.uiSaveConfig`.